frontend/src/bundles/common/helpers/check-url-img-is-valid/check-url-img-valid.ts

```diff
@@ -0,0 +1,7 @@
+const isValidUrlImg = (url: string): boolean => {
```

Comment:
name should start with noun. `validateImageUrl` would be nice, also folder should have the same name as helper itself

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/73a8e608-f875-4be2-83da-faee1808129e)
As for the state dot, what do you suggest?

Reply:
You can create dot with tailwindcss) Simply `rounded-[50%]`
Like this https://play.tailwindcss.com/cRcwJASqdW